src/network/graphql/queries.js

```diff
@@ -337,3 +338,37 @@ export async function searchTopics(app, phrase, filters, orderBy) {
   const result = await request.execute();
   return map(result, (topic) => new Topic(topic, app));
 }
+
+/**
+ * Search for Topics that contains phrase either in title or result setter address
+ * @param {Array} filters Array of objects for filtering. ie. [{ status: 'WAITRESULT' }, { status: 'OPENRESULTSET' }]
+ * @param {Object} orderBy Object with order by fields. ie. { field: 'blockNum', direction: 'DESC' }
+ * @return {Promise} Search result from graphql
```

Comment:
wrong desc for return

Reply:
Oh I meant to change those in the last step before push my final version